sway/commands/input.c

```diff
@@ -24,7 +24,11 @@ struct cmd_results *cmd_input(int argc, char **argv) {
 	char **argv_new = argv+2;
 
 	struct cmd_results *res;
+	struct input_config *old_input_config = current_input_config;
 	current_input_config = new_input_config(argv[0]);
+	if (!current_input_config) {
+		return cmd_results_new(CMD_FAILURE, NULL, "Couldn't allocate config");
```

Comment:
If we can't allocate an input config, we probably shouldn't allocate a command result.

Reply:
Yeah, I thought about that, but I have no idea how to return something else? We can't return something on the stack or a null..

Reply:
This is an unsolved problem bigger than this PR.